### ISSUE TITLE
Compass for development

### ISSUE THREAD

**Issue by alycit** [Devbootcamp/mentor-pairing#113]
I was pulling this down so I can work on it and I was running into some errors.  Readme says to use compass for CSS changes, of which I may have to make a minor few.

I keep getting this error:

```
$ bundle exec compass watch
No application integration exists for rails
```
I have explored it and find there is an issue with compass-rails gem and rails 4 that might be causing this.  I killed compass and compass-rails in the gemfile to force newer versions, and it still didn't help.  

Should I be going down a different path?

@ltw @redsquirrel Help?

**Comment by alycit** [comment]
I really want to work on this today... I can focus on non-pretty things now until answered.  But I need an answer asap.

**Comment by rmw** [comment]
@alycit gem `compass-rails` is included in the [Gemfile](https://github.com/Devbootcamp/mentor-pairing/blob/master/Gemfile). It handles the compass watch.  Compass is already included in [screen.css.scss](https://github.com/Devbootcamp/mentor-pairing/blob/master/app/assets/stylesheets/screen.css.scss).  You don't need to do anything to use compass methods in sass/scss files.

**Comment by alycit** [comment]
Well.. then the readme needs updated.  It states it as a dependency. That's
the only reason I was asking as it did not work for me to set up compass
watch.

On Mon, Feb 9, 2015 at 5:02 PM, Rebecca Miller-Webster <
notifications@github.com> wrote:

> Closed #113 <https://github.com/Devbootcamp/mentor-pairing/issues/113>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Devbootcamp/mentor-pairing/issues/113#event-232845825>
> .
>



-- 

Alyssa Diaz
614-208-1476
@alycit
Instructor, Dev Bootcamp <http://www.devbootcamp.com>
